libsolidity/interface/CompilerStack.cpp

```diff
@@ -59,6 +59,8 @@ CompilerStack::CompilerStack(ReadFileCallback const& _readFile):
 void CompilerStack::setRemappings(vector<string> const& _remappings)
```

Comment:
Now I don't see if `_remappings` contains the names of the modules or the paths of the modules (I will see).